### ISSUE TITLE
JG: No alt text for included graphics (converted from PDF to PNG)

### ISSUE THREAD

**Issue by hturner** [Abhi-1U/texor#59]
Comment from @ajrgodfrey

> There is no `alt` attribute for `<img>` tags when a PDF included with `\includegraphics` has been converted to an `<img>` tag including a PNG. In this case there should be a default alt text, e.g. `alt = "graphic without alt text"`.
> 
> Example: RJ-2009-015
>
> Also: is it necessary to use `<div class="center">` here, or could this be done with CSS on the image?

**Comment by Abhi-1U** [comment]
Given, existing figures are setting `alt-text` as captions or `image` would it be better if all legacy articles left to be converted to have `alt = "graphic without alt text"` text as the default value, unless it is changed in the Rmd? 
I can set the attribute for this in the `texor` package as a default value,  it will override the caption being a part of the alt text as well in this case.

**Comment by Abhi-1U** [comment]
Try out this commit https://github.com/Abhi-1U/texor/commit/31c3714e0ef0194c4dc9c26c782709b0987e2509 on the supplement material of figures.

**Comment by hturner** [comment]
Your commit may also fix https://github.com/Abhi-1U/texor/issues/64?
